api/internal/target/errmissingkustomization.go

```diff
@@ -23,12 +23,7 @@ func (e *errMissingKustomization) Error() string {
 }
 
 func IsMissingKustomizationFileError(err error) bool {
-	_, ok := err.(*errMissingKustomization)
-	if ok {
-		return true
-	}
-	_, ok = errors.Cause(err).(*errMissingKustomization)
-	return ok
```

Comment:
After reading this: https://go.dev/blog/go1.13-errors I think this is a suitable replacement but would appreciate feedback. (see other identical replacements)

Reply:
I'm always needing to re-read that post myself! I think we actually need `As` for these, because we want to match any error of type `errMissingKustomization`, whereas `Is` uses equality. With equality, any struct field differences won't match--e.g. in this case, we won't match errors that have the path field set.

Reply:
Oh yeah, you're right!

> The errors.Is function compares an error to a value.
> The As function tests whether an error is a specific type.g